experiments/AMIP/modular/components/ocean/eisenman_seaice.jl

```diff
@@ -0,0 +1,293 @@
+import ClimaCoupler: FluxCalculator
+import ClimaTimeSteppers as CTS
+import DiffEqBase: ODEProblem, init, step!
+import ClimaCoupler.FluxCalculator: update_turbulent_fluxes_point!, differentiate_turbulent_fluxes!
+import ClimaCoupler.Interfacer: get_field, update_field!
+
+# TODO: move to ClimaCore
+import Base.getindex
+Base.getindex(f::Fields.FieldVector, colidx) =
+    Fields.FieldVector(; zip(propertynames(f), map(x -> getproperty(f, x)[colidx], propertynames(f)))...)
+
+"""
+    EisenmanIceSimulation{P, Y, D, I}
+
+Thermodynamic 0-layer, based on the Semtner 1979 model and later refined by  sea ice model as specified by
+Eisenmen 2009 and Zhang et al 2021.
+"""
+struct EisenmanIceSimulation{P, Y, D, I} <: SeaIceModelSimulation
+    params_ice::P
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+name(::EisenmanIceSimulation) = "EisenmanIceSimulation"
+
+Base.@kwdef struct EisenmanIceParameters{FT <: AbstractFloat}
+    h::FT = 2                       # sea ice height [m]
+    ρ::FT = 1500                    # sea ice density [kg / m3]
+    c::FT = 800                     # sea ice heat capacity [J / kg / K]
```

Comment:
I don't think this parameter enters into the slab sea ice model, at least not if L_ice is assumed constant

Reply:
see above.